decorator.rb

```diff
@@ -0,0 +1,25 @@
+require_relative 'nameable'
+
+class Decorator
+  include Nameable
+
+  def initialize(nameable)
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable.correct_name
+  end
+end
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    super.capitalize
```

Comment:
- [ ] You should use `@nameable.correct_name.capitalize` to capitalize the first letter of the name, kindly consider adjusting your code to not use the `super.capitalize`